.github/workflows/deploy-lts-prow.yaml

```diff
@@ -65,12 +65,17 @@ jobs:
       - name: 'Create GitHub Token secrets'
         run: |
           echo "${{ secrets.APP_PRIVATE_KEY }}" > cert.pem
-          kubectl delete secret github-token -n prow || echo "no existing secret to clean up in namespace prow"
-          kubectl create secret generic github-token -n prow --from-file=cert=cert.pem --from-literal=appid=$GITHUB_APP_ID
-          kubectl delete secret github-token -n test-pods || echo "no existing secret to clean up in namespace test-pods"
-          kubectl create secret generic github-token -n test-pods --from-file=cert=cert.pem --from-literal=appid=$GITHUB_APP_ID
+          kubectl create secret generic github-token -n prow --from-file=cert=cert.pem --from-literal=appid=$GITHUB_APP_ID -o yaml --dry-run=client | kubectl apply -f -
+          kubectl create secret generic github-token -n test-pods --from-file=cert=cert.pem --from-literal=appid=$GITHUB_APP_ID -o yaml --dry-run=client | kubectl apply -f -
           rm cert.pem
 
+      - name: 'Create job configs'
+        run: |
+          envsubst < config/prow/release-branch-jobs/base.yaml > cm.yaml
+          envsubst < config/prow/release-branch-jobs/1.24.yaml >> cm.yaml
```

Comment:
over time we'll add other release branch jobs here, or just loop over all the files